akka-cluster/src/multi-jvm/scala/akka/cluster/ClusterDeathWatchSpec.scala

```diff
@@ -292,10 +304,10 @@ abstract class ClusterDeathWatchSpec
         runOn(first) {
           // fourth system will be shutdown, remove to not participate in barriers any more
           testConductor.shutdown(fourth).await
-          expectMsg(EndActor.End)
+          awaitAssert(expectMsg(EndActor.End)) // get a Unit back here for some reason
```

Comment:
Didn't figure this one out yet, something sends it a `()` before the `EndActor.End` now, but who and why?